plugins/tiddlywiki/multiwikiserver/modules/mws-server.js

```diff
@@ -34,6 +34,7 @@ function Server(options) {
 	this.authenticators = options.authenticators || [];
 	this.wiki = options.wiki;
 	this.boot = options.boot || $tw.boot;
+	this.sqlTiddlerDatabase = $tw.mws.store.sqlTiddlerDatabase;
```

Comment:
It would be better to make the database be one of the options properties, and to amend the call sites to explicitly pass the database:

``
 	this.sqlTiddlerDatabase = options.sqlTiddlerDatabase || $tw.mws.store.sqlTiddlerDatabase;
```